php/php.editor/src/org/netbeans/modules/php/editor/model/impl/ModelBuilder.java

```diff
@@ -86,7 +86,7 @@ class ModelBuilder {
     NamespaceScope build(NamespaceDeclaration node, OccurenceBuilder occurencesBuilder) {
         final NamespaceDeclarationInfo info = NamespaceDeclarationInfo.create(node);
 
-        NamespaceScopeImpl nScope = (info.isDefaultNamespace()) ? defaultNamespaceScope : ModelElementFactory.create(info, this);
+        NamespaceScopeImpl nScope = /*(info.isDefaultNamespace()) ? defaultNamespaceScope :*/ ModelElementFactory.create(info, this);
```

Comment:
What is the reason for the comment-out?

Reply:
This is actually the fix for multiple default namespaces in navigator and for proper default namespaces OffsetRange. I left it commented only if in a near feature something will go wrong with this to give a clue to someone who will debug this. Should I remove the comment or do you need more deep explanation?

Reply:
If you want it commented out, please, leave a comment as the reason because people other than you don't know why. Thanks!